fvtest/tril/tril/jitbuilder_compiler.cpp

```diff
@@ -40,6 +40,12 @@ int32_t Tril::JitBuilderCompiler::compile() {
    return compileWithVerifier(NULL);
 }
 
+#if __cplusplus <= 199711L // Check for C++11 compiler support.
```

Comment:
Given the way the rest of this change looks, I'd just pull this, and use the lambda version universally.

Reply:
Sorry, but did you mean the 'version *without the lambda universally'? (Lambdas were again only added since C++11).

Reply:
I'm cool with this.